docs/contributing/releasing-a-new-version.md

```diff
@@ -0,0 +1,42 @@
+# Releasing a new version
+
+For releasing a new version, we are leveraging `release` command from [symplify/monorepo-builder](https://github.com/Symplify/MonorepoBuilder) package.
+
+All the source codes and configuration of our release process can be found in `utils/releaser` folder that is located in the root of the monorepo.
+
+Each step of the release process is defined as an implementation of  `Symplify\MonorepoBuilder\Release\Contract\ReleaseWorker\ReleaseWorkerInterface`.
+
+## Stages
+
+The whole release process is divided into 3 stages that are run separately:
+
+1. `release-candidate`
+    - the workers are defined in `src/ReleaseWorker/ReleaseCandidate` folder
+    - include steps that are done before the release candidate branch is sent to code review and testing
+1. `release`
+    - the workers are defined in `src/ReleaseWorker/Release` folder
+    - include steps that are done during the actual release
+1. `after-release`
+    - the workers are defined in `src/ReleaseWorker/AfterRelease` folder
+    - include steps that are done after the release
+
+
+## Release command
+
+To perform all the steps of the desired stage, run the following command and follow instructions.
+```
+vendor/bin/monorepo-builder release <release-number> --stage <stage> -v
+```
+If you want only to display all the steps of a particular stage, along with the release worker class names, add the `--dry-run` argument:
```

Comment:
also "If you want only to display a particular stage..." (without all the steps)